diff --git a/node_modules/monaco-plugin-helpers/lib/packageESM.js b/node_modules/monaco-plugin-helpers/lib/packageESM.js
index 3c333a4..ebec9d2 100644
--- a/node_modules/monaco-plugin-helpers/lib/packageESM.js
+++ b/node_modules/monaco-plugin-helpers/lib/packageESM.js
@@ -79,7 +79,7 @@ function _packageESM(options) {
     while (queue.length > 0) {
         var filePath = queue.shift();
         var fileContents = fs.readFileSync(filePath).toString();
-        var info = ts.preProcessFile(fileContents);
+        var info = ts.preProcessFile(fileContents,true,true);
         for (var i = info.importedFiles.length - 1; i >= 0; i--) {
             var importText = info.importedFiles[i].fileName;
             if (shouldSkipImport(importText)) {
